conanfile.txt

```diff
@@ -1,5 +1,5 @@
 [requires]
-xrpl/1.12.0
+xrpl/2.0.0-b4
```

Comment:
This will need to be updated once https://github.com/XRPLF/rippled/pull/4831 is merged.

Reply:
It should be ok to do that in a separate/new PR, right? 

Feel free to create an issue here so that it doesn't get forgotten.
- https://github.com/ripple/validator-keys-tool/issues/new

Reply:
Since XRPLF/rippled#4831 needs to be merged first anyway, I think it would make more sense to do that merge, update the dependency on this PR, and then merge this PR. It wouldn't need another review (or no more than a quick glance), since it's just a version number.